chapters/revisions.tex

```diff
@@ -1,3070 +1,16 @@
 \chapter{Modelica Revision History}\label{modelica-revision-history}
 
-This appendix describes the history of the Modelica Language Design, and its contributors.
+This appendix described the history of the Modelica Language Design, and its contributors.
 This appendix is just present for historic reasons and is not normative.
 The current version of this document is available from \url{https://modelica.org/documents}.
+All versions of this document, and the list of changes is available at \url{https://specification.modelica.org}.
```

Comment:
The table on specification.modelica.org does not contain the changes information like proposed in this PR. Should/will it be update to match the one on README.md? If not, wouldn't it be better to link to README.md?

Reply:
Merging this PR will update that table!
Basically specification.modelica.org's main page is automatically built from README.md of master